package.json

```diff
@@ -22,6 +22,7 @@
     "@babel/core": "^7.4.3",
     "@babel/node": "^7.4.5",
     "@babel/plugin-proposal-class-properties": "^7.4.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
```

Comment:
Is there any indication when the optional chaining plugin will move from being just a proposal? (Granted we're already using the class properties proposal but I'm not familiar with the actual process from moving from a *proposal* state)

Reply:
I'm not sure. It's in stage 1, though some stage 2 to-dos are already done. It's at least mature enough that there's a babel plugin for us that does our heavy lifting.